src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,4 +1,5 @@
-// Add the required props
-export const UserInfo = () => (
-  <>UserInfo markup</>
+export const UserInfo = ({ user }) => (
+  <a className="UserInfo" href={`mailto:${user.email}`}>
```

Comment:
You should add PropTypes to the component to validate the props. This will help debug issues with the types of the values passed in.